### DISCUSSION TITLE
udp input interface

### DISCUSSION THREAD

**Discussion by Cramac** [bluenviron/mediamtx#2739] [Questions]
### Question

Hello. How to specify the interface on which to watch the multicast?
source: udp://vlan300@239.0.1.49:1234

**Comment by aler9** [comment]
Hello, when ingesting a UDP multicast stream into the server, packets are ingested from all available interfaces that support multicast. There's no need to specify the interface.

**Comment by Cramac** [comment]
without specifying an interface, the stream is not visible
not work
ffplay udp://239.0.1.3:1234

work:
ffplay udp://239.0.1.3:1234?localaddr=10.10.30.20

**Reply by aler9** [reply]
This is the question section of MediaMTX, not FFmpeg. MediaMTX doesn't make use of FFmpeg to pull streams, it has a dedicated engine.

**Comment by Cramac** [comment]
this was for example. MediaMTX does not see the stream:
paths:
  test:
    source: udp://239.0.1.3:1234

**Reply by aler9** [reply]
then open an issue and provide details on how to replicate the bug, including server logs and a network dump. Otherwise it's impossible to understand what the problem is.

[a comment by an automated account is not shown]
